pages/common/wami.md

```diff
@@ -0,0 +1,20 @@
+# wami
+
+> A tool that recommends suitable programs for specific tasks. Open-source, easy to use.
+> More information: <https://github.com/evait-security/wami>.
+
+- Search the lake and find expanded results in all categories in descending order:
+
+`wami --show-[a]ll --[S]ort desc --[s]earch-all {{search_string}}`
+
+- Search GitHub to find expanded results, displayed in descending order:
+
+`wami --show-[a]ll --[S]ort desc --github {{search_string}}`
+
+- Search GitHub for topics that match the search string:
+
+`wami --list-topics {{search_string}}`
+
+- Search the lake for a tool used in pentests to query for default credentials and order desc to get the first match at the bottom:
```

Comment:
IMO, the description is a little bit too long (although it does not exceed the [maximum line length](https://github.com/tldr-pages/tldr/blob/main/.markdownlintrc#L5)). What do you guys think? 

```suggestion
- Search the lake for a tool used in pentests to query for default credentials and [S]ort the results in descending order:
```